modules/preview-react/table/lib/TableHeader.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import {Grid, GridProps} from '@workday/canvas-kit-react/layout';
+import {colors} from '@workday/canvas-kit-react/tokens';
+import {createComponent} from '@workday/canvas-kit-react/common';
+
+export const TableHeader = createComponent('th')({
+  displayName: 'Table.Header',
+  Component: ({children, ...elemProps}: GridProps, ref, Element) => {
+    return (
+      <Grid
+        as={Element}
+        ref={ref}
+        alignItems="center"
+        backgroundColor="frenchVanilla100"
+        borderBottom={`1px solid ${colors.soap400}`}
+        justifyContent="left"
+        minHeight="48px"
+        paddingY="xxs"
+        paddingX="s"
+        textAlign="left"
```

Comment:
same thing, I don't think you need textAlign left

Reply:
Correct. Updated.